aws_cloudtrail.tf

```diff
@@ -6,28 +6,29 @@ resource "aws_cloudtrail" "missing-multi-region" {
   name                          = "tf-trail-foobar"
   s3_bucket_name                = "some-s3-bucket"
   s3_key_prefix                 = "prefix"
-  include_global_service_events = false
+  include_global_service_events = true
 }
 
 resource "aws_cloudtrail" "false-multi-region" {
   name                          = "tf-trail-foobar"
   s3_bucket_name                = "some-s3-bucket"
   s3_key_prefix                 = "prefix"
-  include_global_service_events = false
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure multi-regions trail exists for each AWS CloudTrail**
 
**Vulnerability**
Cloud Trail Multi Region not enabled
 
**Remediation**
The AWS API call history produced by CloudTrail enables security analysis, resource change tracking, and compliance auditing. Enable multi-region trail. Refer to control 2.1 in the CIS Amazon Web Services Foundations Benchmark

source: accurics-